src/build/languages.js

```diff
@@ -1,5 +1,4 @@
-module.exports = {
+exports.languages = {
```

Comment:
Sorry @JimiC but I still don't get the rationale here. Could you ellaborate this change?